ajnyga/embedHtmlArticleGalley
# Make compatible with OJS 3.4

Adds missing classes and modifies functions to allow plugin to integrate with OJS 3.4.

To Test:

* Enable the plugin.
* Browse to an article galley and verify it displays correctly.
